content/en/cloud-computing.md

```diff
@@ -8,7 +8,15 @@ tags: ["infrastructure", "fundamental", ""]
 ## What it is
 
 Cloud computing offers compute resources like CPU, network, and disk capabilities on-demand over the internet, allowing users to access and use computing power in a remote physical location. 
-We generally differentiate between private and public cloud, depending on whether the cloud infrastructure is exclusively dedicated to an organization or shared for open public services.
+It extends beyond merely providing resources and we generally differentiate between private and public cloud. These models differ based on the usage and accessibility of the cloud infrastructure.
+
+In a private cloud, the infrastructure is exclusively dedicated to a single organization, and it can exist both on-premise (within the organization's physical location) or off-premise (hosted by a third-party service provider). This setup offers more control over the infrastructure, data security, and regulatory compliance, and can be customized to meet specific business needs.
+
+On the other hand, a public cloud is a shared infrastructure available for open public services, where resources are owned and operated by third-party cloud service providers. 
+
+Additionally, a hybrid cloud model combines aspects of both private and public clouds. This model enables businesses to leverage the advantages of both deployment models by integrating their private cloud with the public cloud. This combination can be used for creating a balanced, flexible, and cost-effective cloud environment.
+
+Therefore, the distinction among cloud computing models isn't solely about whether the infrastructure is private or public. It also involves the location of the infrastructure and the customization level it can offer, as well as the possibility to create a hybrid solution that maximizes the benefits of both environments.
```

Comment:
```suggestion
Therefore, the distinction among cloud computing models isn't solely about whether the infrastructure is private or public. 
It also involves the location of the infrastructure and the customization level it can offer, 
as well as the possibility to create a hybrid solution that maximizes the benefits of both environments.
```